src/main/java/com/google/api/codegen/ruby/RubyProvider.java

```diff
@@ -38,24 +31,6 @@
   static final String SNIPPET_RESOURCE_ROOT =
       RubyGapicProvider.class.getPackage().getName().replace('.', '/');
 
-  public <Element> void output(
-      String packageRoot, String outputPath, Multimap<Element, GeneratedResult> elements)
-      throws IOException {
-    Map<String, Doc> files = new LinkedHashMap<>();
-    for (Map.Entry<Element, GeneratedResult> entry : elements.entries()) {
-      Element element = entry.getKey();
-      GeneratedResult generatedResult = entry.getValue();
-      String root;
-      if (element instanceof Method) {
-        root = ((Method) element).getFile().getFullName().replace('.', '/');
```

Comment:
So, hilarious note: the Method that is imported here is com.google.api.tools.framework.model.Method, but the Method that is provided from RubyDiscoveryProvider is com.google.protobuf.Method. Consequently, this codepath can't be invoked and is thus dead.